src/REPLICA/fix_pimd_langevin.cpp

```diff
@@ -0,0 +1,1497 @@
+      if (temp < 0.0) error->universe_all(FLERR, "Invalid temp value for fix pimd/langevin");
+    } else if (strcmp(arg[i], "lj") == 0) {
+      lj_epsilon = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      lj_sigma = utils::numeric(FLERR, arg[i + 2], false, lmp);
+      lj_mass = utils::numeric(FLERR, arg[i + 3], false, lmp);
+      other_planck = utils::numeric(FLERR, arg[i + 4], false, lmp);
+      other_mvv2e = utils::numeric(FLERR, arg[i + 5], false, lmp);
+      i += 4;
+    } else if (strcmp(arg[i], "thermostat") == 0) {
+      if (strcmp(arg[i + 1], "PILE_L") == 0) {
+        thermostat = PILE_L;
+        seed = utils::inumeric(FLERR, arg[i + 2], false, lmp);
+        i++;
+      }
+    } else if (strcmp(arg[i], "tau") == 0) {
+      tau = utils::numeric(FLERR, arg[i + 1], false, lmp);
+    } else if (strcmp(arg[i], "barostat") == 0) {
+      if (strcmp(arg[i + 1], "MTTK") == 0) {
+        barostat = MTTK;
+      } else if (strcmp(arg[i + 1], "BZP") == 0) {
+        barostat = BZP;
+      } else
+        error->universe_all(FLERR, "Unknown barostat parameter for fix pimd/langevin");
+    } else if (strcmp(arg[i], "iso") == 0) {
+      pstyle = ISO;
+      p_flag[0] = p_flag[1] = p_flag[2] = 1;
+      Pext = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      p_target[0] = p_target[1] = p_target[2] = Pext;
+      pdim = 3;
+    } else if (strcmp(arg[i], "aniso") == 0) {
+      pstyle = ANISO;
+      p_flag[0] = p_flag[1] = p_flag[2] = 1;
+      Pext = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      p_target[0] = p_target[1] = p_target[2] = Pext;
+      pdim = 3;
+    } else if (strcmp(arg[i], "x") == 0) {
+      pstyle = ANISO;
+      p_flag[0] = 1;
+      p_target[0] = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      pdim++;
+    } else if (strcmp(arg[i], "y") == 0) {
+      pstyle = ANISO;
+      p_flag[1] = 1;
+      p_target[1] = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      pdim++;
+    } else if (strcmp(arg[i], "z") == 0) {
+      pstyle = ANISO;
+      p_flag[2] = 1;
+      p_target[2] = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      pdim++;
+    } else if (strcmp(arg[i], "taup") == 0) {
+      tau_p = utils::numeric(FLERR, arg[i + 1], false, lmp);
+      if (tau_p <= 0.0) error->universe_all(FLERR, "Invalid tau_p value for fix pimd/langevin");
+    } else if (strcmp(arg[i], "fixcom") == 0) {
+      if (strcmp(arg[i + 1], "yes") == 0)
+        removecomflag = 1;
+      else if (strcmp(arg[i + 1], "no") == 0)
+        removecomflag = 0;
+    } else {
+      error->universe_all(FLERR, fmt::format("Unknown keyword {} for fix {}", arg[i], style));
+    }
+  }
+
+  /* Initiation */
+
+  global_freq = 1;
+  vector_flag = 1;
+  if (!pstat_flag)
+    size_vector = 10;
+  else if (pstat_flag) {
+    if (pstyle == ISO) {
+      size_vector = 15;
+    } else if (pstyle == ANISO) {
+      size_vector = 17;
+    }
+  }
+  extvector = 1;
+  kBT = force->boltz * temp;
```

Comment:
I've looked through other parts of LAMMPS and the most common variable name for (fix nh, fix rigid) storing `k_B * T` is `kt`. In general, we try to reserve mixed capitalization for classes (but using "CamelCase" conventions), lower case for functions and variables and all caps for constants.

Reply:
solved by [5ff6fd9](https://github.com/lammps/lammps/pull/3660/commits/5ff6fd9aad41084521a1a48381c6ebba6f584426)